account_fiscal_year_closing/README.rst

```diff
@@ -0,0 +1,123 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Fiscal year closing
+===================
+
+This module implements a generic fiscal year closing system for those
+countries where closing/opening moves or other kind of closing operations are
+mandatory in accounting books.
+
+It includes a template mechanism that can be used in localizations for
+providing the possible configurations to be used by the closing engine.
+
+Configuration
+=============
+
+If you want to add a closing template or check existing ones:
+
+#. Go to *Accounting > Configuration > Fiscal year closing > Closing templates*.
+#. Click on Create.
+#. Put a name for the template.
+#. Mark if you want to check if there's any draft move in the date range
+   before making the closing operations.
+#. Select the chart templates for which companies that have this chart you want
+   this template to be available for.
+#. Add one line for each of the journal entries you want to create on the
+   closing operation.
+#. This line has a name, a code, an optional journal (selectable per company),
+   a sequence for ordering this line with the others, and the type we want to
+   assign on the resulting journal entry created.
+#. Then you can configure accounts for being mapped on the section
+   "Accounts mapping".
+#. If you put a destination account on each mapping line, then the balance of
+   the source account will be transferred to that account.
+#. If there's no destination account, then the balance of the account will be
+   nullified through one or several journal items with the opposite balance.
+#. The way these opposite journal items will be created is determined by the
+   closing type in the section "Account closing types":
+   * Balance: There will be only one journal item with the opposite balance.
+   * Un-reconciled: The opposite balance will be grouped by the partner field
+     in the period journal entries.
+#. There's a default closing type to use, and you can specify others by
+   account type.
+#. You can configure a closing operation for being the reverse move of another
+   previous move generated by a closing operation (typically, the opening
+   move after the closing one). For making that, fill the "Inverse config"
+   field with the code of the closing operation you want to reverse, and
+   place this operation after that one (not necessarily immediately after).
+#. If you also want to reconcile the balances of the inverted journal items and
+   the initial ones, click on the check "Reconcile".
+
+You can check l10n_es_account_fiscal_year_closing in OCA/l10n-spain for several
+examples of closing templates.
+
+Usage
+=====
+
+For closing a fiscal year:
+
+#. Go to *Accounting > Adviser > Fiscal year closings*
```

Comment:
*Accounting > Configuration > Fiscal Year Closing*

Reply:
No, this is for the usage, not for the configuration.